lib/data/monorepo.json

```diff
@@ -382,7 +382,7 @@
     "jna": "https://github.com/java-native-access/jna",
     "json-smart-v2": "https://github.com/netplex/json-smart-v2",
     "jsplumb": "https://github.com/jsplumb/jsplumb",
-    "junit5": "https://github.com/junit-team/junit5",
+    "junit-framework": "https://github.com/junit-team/junit-framework",
```

Comment:
add also the old url, so old versions are still properly grouped.

Reply:
Done in 3063e593d0df545bbe3f4e7afa8f213d4801e258